File: examples/OmeroClients/primitives.java
public class primitives {
    public static void main(String[] args) {
        omero.RString a = omero.rtypes.rstring("value");
        omero.RBool b = omero.rtypes.rbool(true);
        omero.RLong l = omero.rtypes.rlong(1l);
        omero.RInt i = omero.rtypes.rint(1);
    }
}

File: examples/OmeroClients/primitives.cpp
#include <omero/RTypesI.h>
using namespace omero::rtypes;
int main() {
    omero::RStringPtr s = rstring("value");
    omero::RBoolPtr b = rbool(true);
    omero::RLongPtr l = rlong(1);
    omero::RIntPtr i = rint(1);
}
